src/Momento.Sdk/ISimpleCacheClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace Momento.Sdk
+{
+    /// <summary>
+    /// Minimum viable functionality of a cache client.
+    ///
+    /// Includes control operations and data operations.
+    /// </summary>
+    [Obsolete("This interface has been renamed to ICacheClient, and is now deprecated")]
+    public interface ISimpleCacheClient : ICacheClient
+    {
+    }
+}
```

Comment:
Dropped a newline here

Reply:
fixed in https://github.com/momentohq/client-sdk-dotnet/pull/408/commits/892832c34379538132c433e8d20e65facf04d56b